example/contentDefinition.json

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
Let's name this file ContentTypeDefinition.json, as it's called in flotiq docs.

Reply:
done